Rakefile

```diff
@@ -82,6 +82,13 @@ task :cent_5rel_dock do
   sh "RS_SET=centos-7-docker INSTALL_METHOD=rpm RPM_REPO='opendaylight-5-release' bundle exec rake beaker"
 end
 
+# Deb VMs
+
+desc "Beaker tests against Ubuntu 16.04 VM with Boron release Deb"
+task :ubuntu_5rel_vm do
+  sh "RS_SET=ubuntu-16 INSTALL_METHOD=deb DEB_REPO='' bundle exec rake beaker"
```

Comment:
I am not sure of the value of DEB_REPO. Should it be 'http://download.opensuse.org/repositories/home:/akshitajha/xUbuntu_16.04/' or just 'opendaylight.list'. Need to figure that out.